hostinfo/filepermissions.lua

```diff
@@ -0,0 +1,102 @@
+--[[
+Copyright 2014 Rackspace
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+--]]
+local HostInfo = require('./base').HostInfo
+
+local forEachLimit = require('async').forEachLimit
+local table = require('table')
+local os = require('os')
+local spawn = require('childprocess').spawn
+
+--[[ Filepermissions Variables ]]--
+local Info = HostInfo:extend()
+function Info:initialize()
+  HostInfo.initialize(self)
+end
+
+function Info:run(callback)
+  if os.type() ~= 'Linux' then
+    self._error = 'Unsupported OS for Filepermissions'
+    callback()
+    return
+  end
+
+  local obj = {}
+
+  local files = {
```

Comment:
shouldn't this be a check parameter, the list of files to check?

If we are going to hard code it, right now this feels like an arbitrary list, is there reasoning we can document here? Like this is the list as of $DATE from an IDS system we trust/like?